flink-runtime/src/test/java/org/apache/flink/runtime/state/StateBackendTestBase.java

```diff
@@ -152,17 +137,16 @@
  * <p>NOTE: Please ensure to close and dispose any created keyed state backend in tests.
  */
 @SuppressWarnings("serial")
-public abstract class StateBackendTestBase<B extends AbstractStateBackend> extends TestLogger {
+@ExtendWith({ParameterizedTestExtension.class, TestLoggerExtension.class})
```

Comment:
No, I mean not `TestLoggerExtension`
Using the `META-INF/services/org.junit.jupiter.api.extension.Extension` is the better approach here.
for more details have a look at
https://github.com/apache/flink/commit/ecf67c67dbcf633c4533f8cb719f32512c19d0ce#diff-d96a3b8d09f526305595964a0010cf113e59999006534653bf45b9971b66241f

Reply:
Thank you for your advice.

However, I find that `META-INF/services/org.junit.jupiter.api.extension.Extension` already exists in `flink-runtime/src/test/resources/` directory, and the content in the file is:
```
org.apache.flink.util.TestLoggerExtension
```

I think the `TestLoggerExtension.class` I used here is the `org.apache.flink.util.TestLoggerExtension` in the `flink-runtime/src/test/resources/META-INF/services/org.junit.jupiter.api.extension.Extension`